packages/katello/katello/katello-remove

```diff
@@ -3,19 +3,29 @@
 require 'optparse'
 require 'fileutils'
 
+options = {}
 optparse = OptionParser.new do |opts|
-  opts.banner = "This script is used to remove all katello related packages and services. \nThis should only be used if you are sure you are wanting to completly remove Katello from this machine and lose all of your settings and synced content."
+  opts.banner = "This script is used to remove all Katello related packages and services. \nThis should only be used if you are sure you are wanting to completly remove Katello from this machine and lose all of your settings and synced content."
 
+  options[:unattended] = false
```

Comment:
You actually don't need this as calling a `options[:unattended]` when the key doesn't exist will return nil, and nil can be used in boolean conditions as false